docs/reference/operations/index.md

```diff
@@ -0,0 +1,30 @@
+# Operations
+
+An _operation_ queries or updates the database state.
+
+There are four operations available to clients:
+
+- [`create_accounts`](./create_accounts.md): create `Account`s
+- [`create_transfers`](./create_transfers.md): create `Transfer`s
+- [`lookup_accounts`](./lookup_accounts.md): fetch `Account`s by `id`
+- [`lookup_transfers`](./lookup_transfers.md): fetch `Transfer`s by `id`
+
+## Events and Results
+
+Each operation has a corresponding _event_ and _result_ type.
+
+- The client sends events to the cluster.
+- The client receives results from the cluster (1:1 for each event sent).
+- Events and results for the same operation are
+  [batched](../../usage/integration.md#batching-events) for throughput.
+- A batch of an operation's events is called a [_request_](../../usage/integration.md#requests).
+- A batch of an operation's results is called a _reply_.
+
+Client implementations provide an API for sending events and decoding the corresponding results.
+
+| Operation          | Event                                        | Result                                                 |
+| ------------------ | -------------------------------------------- | ------------------------------------------------------ |
+| `create_accounts`  | [`Account`](./create_accounts.md#Event)      | [`CreateAccountResult`](./create_accounts.md#Result)   |
+| `create_transfers` | [`Transfer`](./create_transfers.md#Event)    | [`CreateTransferResult`](./create_transfers.md#Result) |
+| `lookup_accounts`  | [`Account.id`](./lookup_accounts.md#Event)   | [`Account`](./lookup_accounts.md#Result) or nothing    |
```

Comment:
Maybe change Result of the last two to `List of [\`Account\`](...)`?

Reply:
It is a question of terminology. We say that a Result is singular, and a batch of Results is called a Reply.

Reply:
Then maybe change the column headers to say `Event (Batched)` and `Result (Batched)` or something like that since the operation create_accounts/whatnot always deals with lists.

Reply:
I'll change the previous sentence to:

> Client implementations provide an API for sending batched events and decoding the corresponding batched results.

I want to leave the table in the singular (unbatched); it will match better with the API once clients implement batching internally.